tests/techmap/booth.ys

```diff
@@ -10,6 +10,19 @@ endmodule
 EOF
 booth
 sat -verify -set a 0 -set b 0 -prove y 0
+
 design -reset
+test_cell -s 1694091355 -n 100 -script booth_map_script.ys_ $mul
 
-test_cell -s 1694091355 -n 100 -script booth_map_script.ys_ $mul
\ No newline at end of file
+design -reset
+read_verilog <<EOF
+module top(a,b,y);
+input wire [4:0] a;
+input wire [5:0] b;
+output wire [6:0] y;
+assign y = a * b;
+endmodule
+EOF
+synth -run :fine
+# test compatibility with alumacc
```

Comment:
I think that ideally at this point we would assert the presence of `$macc` to make sure we're testing what we want to test